Dockerfile

```diff
@@ -16,6 +16,11 @@ RUN dnf install -y fedora-workstation-repositories dnf-plugins-core ; \
 USER odoo
 RUN pip3 install --user odoorpc "pyyaml<5.1"
 
+USER root
+ADD https://chromedriver.storage.googleapis.com/74.0.3729.6/chromedriver_linux64.zip /bin/chromedriver.zip
```

Comment:
Is this fix meant to be temporary, until the dependency mismatch is fixed upstream?

If so, I suggest to add a TODO comment to get rid of it at some point.

Otherwise, setting the version string with an env var (like you do for `ODOO_VERSION`) would be a cleaner solution.

Reply:
Temporary, I think. I'll add a comment.